underreact.config.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const html = require('./html');
+const mapboxAssembly = require('@mapbox/mbx-assembly');
+
+module.exports = () => {
+  return {
+    siteBasePath: 'svg-to-geojson',
+    browserslist: mapboxAssembly.browsersList,
```

Comment:
🤔 is this needed?

Reply:
No strong downside except that it saves from adding `@mapbox/mbx-assembly` as a dependency if we were adding those through HTML instead. I'm fine to keep this as it is though.